src/integrationTest/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerRemoveStayIT.java

```diff
@@ -0,0 +1,271 @@
+            .stayReason(StayReason.OTHER)
+            .additionalDetail("additional detail")
+            .isCaseStayed(YesOrNo.YES)
+            .build();
+
+        final CaseData caseData = CaseData.builder()
+            .caseStay(caseStay)
+            .build();
+
+        mockMvc.perform(post(ABOUT_TO_SUBMIT_URL)
+                .contentType(APPLICATION_JSON)
+                .header(SERVICE_AUTHORIZATION, TEST_AUTHORIZATION_TOKEN)
+                .header(AUTHORIZATION, TEST_AUTHORIZATION_TOKEN)
+                .content(objectMapper.writeValueAsString(
+                    callbackRequest(
+                        caseData,
+                        CASEWORKER_REMOVE_STAY,
+                        CaseStayed.toString())))
+                .accept(APPLICATION_JSON))
+            .andExpect(
+                status().isOk())
+            .andExpect(
+                content().json(expectedResponse(CASEWORKER_REMOVE_STAY_ABOUT_TO_SUBMIT_RESPONSE))
+            );
+    }
+
+    @Test
+    void shouldReturnConfirmationMessageIfNotificationsDispatchedOnSubmitted() throws Exception {
+        final CaseData caseData = CaseData.builder()
+            .cicCase(CicCase.builder()
+                .subjectCIC(Set.of(SubjectCIC.SUBJECT))
+                .applicantCIC(Set.of(ApplicantCIC.APPLICANT_CIC))
+                .representativeCIC(Set.of(RepresentativeCIC.REPRESENTATIVE))
+                .notifyPartySubject(Set.of(SUBJECT))
+                .notifyPartyRepresentative(Set.of(REPRESENTATIVE))
+                .notifyPartyApplicant(Set.of(APPLICANT_CIC))
+                .contactPreferenceType(EMAIL)
+                .representativeContactDetailsPreference(EMAIL)
+                .applicantContactDetailsPreference(EMAIL)
+                .fullName("Test Name")
+                .email("test@test.com")
+                .representativeFullName("Rep Name")
+                .representativeEmailAddress("representative@test.com")
+                .applicantFullName("Applicant Name")
+                .applicantEmailAddress("applicant@test.com")
+                .build()
+            )
+            .build();
+
+        String response = mockMvc.perform(post(SUBMITTED_URL)
+                .contentType(APPLICATION_JSON)
+                .header(SERVICE_AUTHORIZATION, TEST_AUTHORIZATION_TOKEN)
+                .header(AUTHORIZATION, TEST_AUTHORIZATION_TOKEN)
+                .content(objectMapper.writeValueAsString(
+                    callbackRequest(
+                        caseData,
+                        CASEWORKER_REMOVE_STAY)))
+                .accept(APPLICATION_JSON))
+            .andExpect(
+                status().isOk())
+            .andReturn()
+            .getResponse()
+            .getContentAsString();
+
+        assertThatJson(response)
+            .inPath(CONFIRMATION_HEADER)
+            .isString()
+            .contains("# Stay Removed from Case \n## A notification has been sent to: Subject, Representative, Applicant");
```

Comment:
Can we verify the send email method was called and how many times. There are examples of this in other tests if you need a reference